packages/strapi-plugin-upload/config/schema.graphql.js

```diff
@@ -57,6 +56,15 @@ module.exports = {
           return await strapi.plugins.upload.services.upload.updateFileInfo(id, info);
         },
       },
+      deleteFile: {
+        description: 'Delete one file',
+        resolverOf: 'plugins::upload.upload.deleteFile',
```

Comment:
you are referencing a controller that doesn't exist :) it would have to be the destroy controller action :)

Reply:
Curious that the unit tests were ok. Fixed!